ui/raidboss/data/07-dt/trial/valigarmanda-ex.ts

```diff
@@ -0,0 +1,760 @@
+      },
+      outputStrings: {
+        startEast: Outputs.getRightAndEast,
+        startWest: Outputs.getLeftAndWest,
+      },
+    },
+    {
+      id: 'Valigarmanda Ex Feather of Ruin',
+      type: 'Ability',
+      netRegex: { id: '8FDE', source: 'Feather of Ruin', capture: false },
+      // only need to capture one, but delay the alert for people to rotate
+      delaySeconds: 5,
+      durationSeconds: 8,
+      suppressSeconds: 99999,
+      infoText: (_data, _matches, output) => output.killFeather!(),
+      outputStrings: {
+        killFeather: {
+          en: 'Kill Feather => Stand in safe tile',
+        },
+      },
+    },
+    {
+      id: 'Valigarmanda Ex Post-Feather Spread',
+      type: 'Ability',
+      // as soon as the feathers explode, people can spread
+      // use a longer duration to better align to the mechanic
+      netRegex: { id: '8FDF', source: 'Valigarmanda', capture: false },
+      durationSeconds: 10,
+      alertText: (data, _matches, output) => {
+        if (data.firstStormDebuff === undefined)
+          return;
+        return output[data.firstStormDebuff]!();
+      },
+      outputStrings: {
+        ice: {
+          en: 'Spread - elevated tile',
+        },
+        lightning: {
+          en: 'Spread - ground tile',
+        },
+      },
+    },
+    {
+      id: 'Valigarmanda Ex Storm Big AOEs + Bait',
+      type: 'StartsUsing',
+      // no cast bar, and baited AOE puddles follow
+      // 8FC5: Susurrant Breath (conal)
+      // 8FC9: Slithering Strike (out)
+      // 8FCD: Strangling Coil (donut)
+      netRegex: { id: ['8FC5', '8FC9', '8FCD'], source: 'Valigarmanda' },
+      durationSeconds: 7,
+      alertText: (_data, matches, output) => {
+        if (matches.id === '8FC5') {
+          return output.combo!({ type: output.cone!() });
+        } else if (matches.id === '8FC9') {
+          return output.combo!({ type: output.out!() });
+        }
+        return output.combo!({ type: output.donut!() });
+      },
+      outputStrings: {
+        ...bigAoeOutputStrings,
+        combo: {
+          en: '${type} => Bait Puddles',
+        },
+      },
+    },
+    {
+      id: 'Valigarmanda Ex Crackling Cataclysm',
+      type: 'StartsUsing',
+      netRegex: { id: '8FC1', source: 'Valigarmanda', capture: false },
+      suppressSeconds: 2,
+      response: Responses.moveAway('alarm'),
+    },
+    {
+      // NOTE: Have not seen any logs with stale position data, but if its an issue,
+      // this can be changed to a `getCombatants` call.
+      id: 'Valigarmanda Ex Storm Arcane Sphere Collect',
+      type: 'AddedCombatant',
+      netRegex: { name: 'Arcane Sphere' },
+      durationSeconds: 6,
+      run: (data, matches) => {
+        const posY = parseFloat(matches.y);
+        // 5 spheres will spawn in 6 possible y positions: 87.5, 92.5, 97.5, 102.5, 107.5, 112.5
+        if (posY < 88)
+          data.arcaneLaneSafe = data.arcaneLaneSafe.filter((l) => l !== 'northFront');
```

Comment:
Minor nit, could we rewrite this as `filter((lane) => lane !== 'northFront'`? In a number of fonts the lowercase L looks astonishingly like a number 1. If we're bumping up against line length, then don't bother, but otherwise it's easier for me and probably others to read a short variable name.